feat(vite-plugin): support different extensions for catalogs. Improve coverage (#1526)
lingui/js-lingui: website/docs/ref/vite-plugin.md

}
```

:::note

> If you use format which has different extension then `*.po` you need to specify `?lingui` suffix

Reviewer: ```suggestion
If you are using a format that has a different extension than `*.po`, you need to specify the `?lingui` suffix:
```
